### PR TITLE
Add creative commons licenses and wizard selector

### LINKED ISSUE #6572: Allow Static Themes packages to be submitted via devhub
[Tracker mozilla/addons/issues/501]

Actually, this should be possible already.  But we need specific support in the views/models to identify and treat these add-ons differently to extensions.

### PR DESCRIPTION
**Pull request by eviljeff**
fixes #6572 
This just finishes the upload flow.  To be addressed in follow-up issues:
* devhub edit listing/versions
* the a create-o-theme wizard, like the existing personas one

### CODE REVIEW

File: src/olympia/devhub/templates/devhub/addons/submit/describe.html [hunk 1]
```diff
@@ -93,12 +94,14 @@
     </div>
     {% if license_form %}
       <b>{{ _('License:') }}</b>
+      {{ license_form.builtin.errors }}
+      {% if addon.type == amo.ADDON_STATICTHEME %}
+        {% include 'devhub/addons/includes/cc_license_chooser.html' %}
+      <div class="addon-submission-field" id="license-list">
+      {% else %}
       <div class="addon-submission-field">
```
Comment (diox) [inline src/olympia/devhub/templates/devhub/addons/submit/describe.html]: I don't have a good solution, but man, the indentation is a mess in this file...

File: src/olympia/devhub/templates/devhub/versions/file_status_message.html [hunk 2]
```diff
@@ -1,4 +1,4 @@
-<strong>{{ _('File {file_id} ({platform})')|format_html(file_id=file_id, platform=platform) }}</strong>
+<strong>{{ _('File {file_id} ({platform})')|format_html(file_id=fileid, platform=platform) }}</strong>
```
Comment (diox) [inline src/olympia/devhub/templates/devhub/versions/file_status_message.html]: Drive-by fix eh?

File: static/js/zamboni/devhub.js [hunk 3]
```diff
@@ -1479,3 +1480,77 @@ function initInAppConfig($dom) {
...
+            // Choose "No" and "No" for the "commercial" and "derivative" questions.
+            $('input[name="cc-noncom"][value=1], input[name="cc-noderiv"][value=2]').prop('checked', true);
+        } else {
+            $('.noncc').removeClass('disabled');
+        }
+    }
+    function setLicenseFromWizard() {
+        cc_data = $('input[name^="cc-"]:checked').map(function() {
+            return this.dataset.cc}).get();
+        radio = $('#submit-describe #license-list input[type=radio][data-cc="'+cc_data.join(' ')+'"]');
```
Comment (diox) [inline static/js/zamboni/devhub.js]: Spaces around `+` here and in the rest of the file

File: src/olympia/devhub/templates/devhub/addons/submit/describe.html (hunk 1 above)
Comment (eviljeff) [reply src/olympia/devhub/templates/devhub/addons/submit/describe.html]: its fixable, just with a lot more diff noise.

File: src/olympia/devhub/templates/devhub/versions/file_status_message.html (hunk 2 above)
Comment (eviljeff) [reply src/olympia/devhub/templates/devhub/versions/file_status_message.html]: 😉 
Its probably been broken since it was written X years ago, and only uncovered after #5837

File: src/olympia/devhub/templates/devhub/addons/submit/describe.html (hunk 1 above)
Comment (eviljeff) [reply src/olympia/devhub/templates/devhub/addons/submit/describe.html]: And there's always the trade-off between indentation with `{% if ... %}` in the template meaning the resulting html is oddly indented.

File: static/js/zamboni/devhub.js (hunk 3 above)
Comment (eviljeff) [reply static/js/zamboni/devhub.js]: https://github.com/mozilla/addons-server/pull/6740/commits/c1995a248a6ef7e288f102b2debd7e0f4dcbd12c

File: src/olympia/devhub/templates/devhub/addons/submit/describe.html (hunk 1 above)
Comment (diox) [reply src/olympia/devhub/templates/devhub/addons/submit/describe.html]: Yeah it's going to look bad regardless, don't worry about it too much

Comment (eviljeff) [reply src/olympia/devhub/templates/devhub/addons/submit/describe.html]: didn't end up being that bad https://github.com/mozilla/addons-server/pull/6740/commits/b8c370c2c5ca9c38f5c6ecf876922a149152f200
